ash-window/Cargo.toml

```diff
@@ -27,4 +27,4 @@ ash = { path = "../ash", version = "0.37", default-features = false, features =
 
 [[example]]
 name = "winit"
-required-features = ["ash/linked"]
+required-features = ["ash/linked", "ash/std"]
```

Comment:
This doesn't make the example work out-of-the-box:

```console
$ cargo r -p ash-window --example winit
error: target `winit` in package `ash-window` requires the features: `ash/linked`, `ash/std`
Consider enabling them by passing, e.g., `--features="ash/linked ash/std"`
```

Just like `"linked"`, add `"std"` to the list of features for the `ash` dependency...

Reply:
Though having ash-window enable `ash/std` would be valid, it would force enable `std` just be depending on ash-window.

I think the proper solution here is to give ash-window a default `std` feature.

ash-window can be no-std capable since raw-window-handle is no-std.

raw-window-metal however requires std which complicates this a bit.

Reply:
> Though having ash-window enable `ash/std` would be valid, it would force enable `std` just be depending on ash-window.

That's why you have to add it to `DEV-dependencies`, next to `"linked"` like I said (and **not** in the `[dependencies]` table like you did now).

However, it'd be great if `ash-window` could be `no_std` capable too.

> raw-window-metal however requires std which complicates this a bit.

Can that be addressed? @msiglreith owns that crate so we can iterate quite easily.

Reply:
I quickly looked into it and seems a bit tricky due to `objc` dependency